webscrapers/BCB_IF/Bacen_IF_Data.py

```diff
@@ -0,0 +1,60 @@
+# Quantitative Finance
+# Insper
+# Authors: Ivan Zemella and Renan Pereira
+
+# Imports BancoCentral class
+from classes.BancoCentral import BancoCentral
+
+# Imports Pandas class
+import pandas as pd
+
+escolha = input('Do you want to enter periods manually? Y / N ')
```

Comment:
I know this is an example routine, but there is no need to ask the user if it should be a manual input or not.